apps/frontend/static_src/scss/components/doc-mode.scss

```diff
@@ -0,0 +1,17 @@
+.doc-mode:hover {
```

Comment:
We should align the name of this component with the streamfield block name. If we use `.doc-mode` the streamfield block must be `DocModeBlock`. If we go for `SectionGridBlock` this must be `.section-grid`

Reply:
:+1: